doc/review-sentences.md

```diff
@@ -0,0 +1,10 @@
+## Review Criteria
+
+### Make sure the sentence meets the following criteria:
+
+1. The sentence must be spelled correctly.
+1. The sentence must be grammatically correct.
+1. The sentence must be speakable.
+1. If the sentence meets the criteria, click the "yes" button on the right.
+1. If the sentence does not meet the above criteria, click the "no" button on the right. If you are unsure about the sentence, you may also skip it and move on to the next one.
```

Comment:
that doesn't really match the icons and swipe interface anymore, but that's the same in the normal how to, so let's change that outside of this PR.

Reply:
Great! Thanks :)